dali/operators/sequence/optical_flow/optical_flow.cc

```diff
@@ -81,6 +82,40 @@ constexpr int kNInputDims = 4;
 
 template<>
 void OpticalFlow<GPUBackend>::RunImpl(Workspace<GPUBackend> &ws) {
+  // This is a workaround for an issue with nvcuvid in drivers >460 where concurrent
+  // use on default context and non-default streams may lead to memory corruption.
+  // TODO(michalz): add an upper bound when the problem is fixed
+  cudaStream_t of_stream = ws.stream();
+#if NVML_ENABLED
+  {
+    static float driver_version = []()->float {
+      char version[80];
+      CUDA_CALL(nvmlInitChecked());
+      CUDA_CALL(nvmlSystemGetDriverVersion(version, sizeof version));
+
+      return std::stof(version);
```

Comment:
Is it guaranteed that the version can always be converted to float? No alpha characters? Or double '.' in the version string?

Reply:
We already do it in two other places.